setup.py

```diff
@@ -45,7 +45,7 @@
     "fs>=2.0",
     "jsonschema[format]>=3.0.1",
     "kombu>=4.6,<4.7",
-    "mock>=3.0",
+    "mock>=3.0,<4",
```

Comment:
Is this for `master` or could it rather be for `maint-0.7` as well?  Thinking about a situation where we shall have to work in four months on some 0.7.x bugfix.... It could be handy to merge to the lowest branch where the fix applies to and propagate upwards only afterwards?

Reply:
Yes, makes sense, I am changing the branch target :)